public/app/features/provisioning/RepositoryStatusPage.tsx

```diff
@@ -74,10 +74,15 @@ export default function RepositoryStatusPage() {
       }}
       actions={
         data && (
-          <Stack>
+          <Stack gap={1}>
```

Comment:
```suggestion
          <Stack>
```
`gap={1}` is default.